consent/sql_helper.go

```diff
@@ -90,6 +90,20 @@ var sqlParamsConsentRequestHandled = []string{
 	"was_used",
 }
 
+var sqlParamsConsentRequestHandledUpdate = []string{
+	"challenge=:challenge",
```

Comment:
Let's avoid duplication and construct this from `sqlParamsConsentRequestHandled` instead.